draft-ietf-tls-dtls-connection-id.md

```diff
@@ -414,20 +440,23 @@ Every identifier introduces the risk of linkability, as explained in {{RFC6973}}
 In addition, endpoints can use the CID to attach arbitrary metadata
 to each record they receive. This may be used as a mechanism to communicate
 per-connection information to on-path observers. There is no straightforward way to
-address this with CIDs that contain arbitrary values; implementations
-concerned about this SHOULD refuse to use connection ids.
+address this concern with CIDs that contain arbitrary values. Implementations
+concerned about this aspects SHOULD refuse to use CIDs.
 
 An on-path adversary, who is able to observe the DTLS protocol exchanges between the
 DTLS client and the DTLS server, is able to link the observed payloads to all
-subsequent payloads carrying the same connection id pair (for bi-directional
+subsequent payloads carrying the same ID pair (for bi-directional
 communication).  Without multi-homing or mobility, the use of the CID
-is not different to the use of the 5-tuple.
+exposes the same information as the 5-tuple.
 
 An on-path adversary can also black-hole traffic or create a reflection attack
```

Comment:
At the risk of repeating it too often:

I would prefer, not to relate the "black-hole" attack of an "on-path-adversary" with the CID!
Until now, nobody presented any reason, why such a general "black-hole" is related to this CID.

So:
- either generalize the threat

"An on-path adversary can no only black-hole traffic, which is generally also possible without the CID, additionally a reflection attack could be created ..."
 
- or remove the "black-hole".

In my experience, such threats, which are for experts obviously not related to a specific technique, will be cited by others, and then it will turn into time consuming discussion instead of spending the time in protection against real threats.

THEREFORE:
Please, don't overstate the threat! The left real threat one is hard enough.

Reply:
Fair enough. That threat is not new to this document and hasn't been a concern to anyone as far as I know. I will remove it.

Reply:
Changed the text and removed the term black-hole.